test/test_powersystemconstructors.jl

```diff
@@ -121,7 +95,7 @@ end
     # with nullable values. Consider adding them so that this "demo-constructor" works.
     # If that isn't appropriate for this type, add it to types_to_skip below.
 
-    types_to_skip = (System, TestDevice, TestRenDevice)
```

Comment:
This was a bug that manifested if you `include` this test file without first including the test where these types are defined (which happens to be later alphabetically).  There should not be a need to skip these types.